src/redux/reducers/initialState.js

```diff
@@ -80,7 +80,9 @@ const initialState = {
     liked: false,
     disliked: false,
     error: ""
-  }
+  },
+  profile: null,
+  profiles: null
```

Comment:
Whats the difference between `profiles` and `profile`?

Reply:
@kimotho-njoki `profiles` is redundant. I will remove it.